include/swift/Basic/JSONSerialization.h

```diff
@@ -481,6 +481,16 @@ struct ScalarTraits<uint32_t> {
   static bool mustQuote(StringRef) { return false; }
 };
 
+#if defined(_MSC_VER)
+// In MSVC, 'unsigned long' is 32bit size and different from uint32_t,
+// and it is used to define swift::sys::ProcessId.
```

Comment:
Yeah, you're right. Understandable that LLVM wouldn't want to include all of windows.h just for `DWORD`.